packages/state-transition/src/epoch/index.ts

```diff
@@ -59,19 +65,53 @@ export function processEpoch(fork: ForkSeq, state: CachedBeaconStateAllForks, ca
     throw new Error("Lodestar does not support this network, parameters don't fit number value inside state.slashings");
   }
 
-  processJustificationAndFinalization(state, cache);
+  {
+    const timer = metrics?.epochTransitionStepTime.startTimer({
```

Comment:
would it make sense to have a metric utility function `withTimer` which just does the instrumentation? could refactor in a separate PR

Reply:
yeah, that would be nice